### PR TITLE
harmonia: one more pass for bind volumes

### PR DESCRIPTION
**Pull request by stevvooe**
## Description

After some discussion with the team, we've added a few more conditions.

## Reviews

- [ ] Technical review
- [ ] Editorial review
- [ ] Product review

### CODE REVIEW

File: content/manuals/harmonia/_index.md [hunk 1]
```diff
@@ -263,7 +263,9 @@ Run `docker harmonia doctor` to print helpful troubleshooting information.
 - KinD does not run on Project Harmonia due to some hard-coded assumptions to ensure it's running in a privileged container. K3d is a good alternative.
 - Containers cannot access host through DNS `host.docker.internal`.
 - File binds (non-directory binds) are currently static, meaning changes will not be reflected until the container is restarted. This also affects Compose configs and secrets directives.
-- Bind mounts, such as `-v /localpath:/incontainer` in the `docker run` command, are not supported without using a file-sync.
+- Bind _mounts_, such as `-v /localpath:/incontainer` in the `docker run` command, require starting a file-sync.
+- Starting a file-sync for a directory with a large amount of may take extra time to sync and become ready for us in a container.
```
Comment (nicksieger) [inline content/manuals/harmonia/_index.md:267]:
```suggestion
- Starting a file-sync for a directory with a large amount of may take extra time to sync and become ready for use in a container.
```

**Review by nicksieger — APPROVED** [review]

File: content/manuals/harmonia/_index.md (hunk 1 above)
Comment (rodnymolina) [reply content/manuals/harmonia/_index.md:267]: "a large amount *of data/files" may take..."

File: content/manuals/harmonia/_index.md [hunk 2]
```diff
@@ -263,7 +263,9 @@ Run `docker harmonia doctor` to print helpful troubleshooting information.
 - KinD does not run on Project Harmonia due to some hard-coded assumptions to ensure it's running in a privileged container. K3d is a good alternative.
 - Containers cannot access host through DNS `host.docker.internal`.
 - File binds (non-directory binds) are currently static, meaning changes will not be reflected until the container is restarted. This also affects Compose configs and secrets directives.
-- Bind mounts, such as `-v /localpath:/incontainer` in the `docker run` command, are not supported without using a file-sync.
+- Bind _mounts_, such as `-v /localpath:/incontainer` in the `docker run` command, require starting a file-sync.
```
Comment (rodnymolina) [inline content/manuals/harmonia/_index.md:266]: s / starting / creating

**Review by rodnymolina — APPROVED** [review]
LGTM

File: content/manuals/harmonia/_index.md (hunk 2 above)
Comment (stevvooe) [inline content/manuals/harmonia/_index.md:266]:
```suggestion
- Bind _mounts_, such as `-v /localpath:/incontainer` in the `docker run` command, require creating a file-sync.
```

File: content/manuals/harmonia/_index.md [hunk 3]
```diff
@@ -263,7 +263,9 @@ Run `docker harmonia doctor` to print helpful troubleshooting information.
 - KinD does not run on Project Harmonia due to some hard-coded assumptions to ensure it's running in a privileged container. K3d is a good alternative.
 - Containers cannot access host through DNS `host.docker.internal`.
 - File binds (non-directory binds) are currently static, meaning changes will not be reflected until the container is restarted. This also affects Compose configs and secrets directives.
-- Bind mounts, such as `-v /localpath:/incontainer` in the `docker run` command, are not supported without using a file-sync.
+- Bind _mounts_, such as `-v /localpath:/incontainer` in the `docker run` command, require creating a file-sync.
+- Starting a file-sync for a directory with a large amount of may take extra time to sync and become ready for use in a container.
```
Comment (dvdksn) [inline content/manuals/harmonia/_index.md:267]:
```suggestion
- Creating a [synchronized file share](/manuals/desktop/features/synchronized-file-sharing.md) for a directory with a large amount of may take extra time to sync and become ready for use in a container.
```

File: content/manuals/harmonia/_index.md [hunk 4]
```diff
@@ -263,7 +263,9 @@ Run `docker harmonia doctor` to print helpful troubleshooting information.
...
+- Bind _volumes_, such as those created with `docker volume create --driver local --opt type=none --opt o=bind --opt device=/some/host/path myvolname` or via the compose equivalent, are not supported.
```
Comment (dvdksn) [inline content/manuals/harmonia/_index.md:268]: nit: `local` is the default driver, so can be omitted in this example for brevity.

```suggestion
- Bind _volumes_, such as those created with `docker volume create --opt type=none --opt o=bind --opt device=/some/host/path myvolname` or via the compose equivalent, are not supported.
```

Comment (stevvooe) [reply content/manuals/harmonia/_index.md:268]: I actually left this in so the context would be clear. Feel free to commit suggestion if you think its not necessary.

**Review by dvdksn — APPROVED** [review]

File: content/manuals/harmonia/_index.md (hunk 1 above)
Comment (rodnymolina) [reply content/manuals/harmonia/_index.md]: @stevvooe, the above suggestion didn't go in...

- Creating a synchronized file share for a directory with a large amount of [**missing noun**] may take extra time to sync and become ready for use in a container.

**Pull request merged**